### PR TITLE
design - add components to Storybook

### PR DESCRIPTION
**Pull request by lorenzo-cavazzi**
You can check Storybook at https://renku-ci-ui-3286.dev.renku.ch/storybook/

/deploy

### CODE REVIEW

File: client/src/storybook/bootstrap/Tooltips.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+};
+
+export const TooltipDisabled_: Story = {
+  render: function TooltipStory({ placement }) {
+    const ref = useRef<HTMLDivElement>(null);
+
+    return (
+      <>
+        <div className="p-5">
+          <div className="d-inline-block" ref={ref}>
```
Comment (leafty) [inline client/src/storybook/bootstrap/Tooltips.stories.tsx:70]: This is missing a `tabindex` to make it selectable by keyboard. See: https://getbootstrap.com/docs/5.3/components/tooltips/#disabled-elements.

**Review by leafty — CHANGES_REQUESTED** [review]

File: client/src/styles/renku_additional_classes.scss
[the remark this answers is not in the document]
Comment (lorenzo-cavazzi) [reply client/src/styles/renku_additional_classes.scss:1]: Removed here https://github.com/SwissDataScienceCenter/renku-ui/pull/3286/commits/02aed62a41523ba865c5adfd5b4402313d7dbd56
I'll open a separate PR with a proper solution.

File: client/src/storybook/bootstrap/BadgeInfo.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+CircleFill.displayName = "CircleFill";
+
+export const BadgeInfo_: Story = {
+  render: (_args) => {
+    return (
+      <Badge color="info" className={cx("border", _args.status)}>
+        {_args.loader ? (
+          <Loader size={12} className={cx("me-1")} inline />
+        ) : (
+          <CircleFill className={cx("me-1")} />
```
[the remark this answers is not in the document]
Comment (lorenzo-cavazzi) [reply client/src/storybook/bootstrap/BadgeInfo.stories.tsx]: Fixed here https://github.com/SwissDataScienceCenter/renku-ui/pull/3286/commits/8a928665a3d551988931766c449cbedb8052119e

File: client/src/storybook/bootstrap/Tooltips.stories.tsx (hunk 1 above)
Comment (lorenzo-cavazzi) [reply client/src/storybook/bootstrap/Tooltips.stories.tsx]: Ha, good catch!
Story updated here https://github.com/SwissDataScienceCenter/renku-ui/pull/3286/commits/9f59c35eb90e56398705c1f8a9f1db27cfb51a1d

File: client/src/styles/renku_additional_classes.scss
[the remark this answers is not in the document]
Comment (lorenzo-cavazzi) [reply client/src/styles/renku_additional_classes.scss:1]: https://github.com/SwissDataScienceCenter/renku-ui/pull/3327

[the remark this answers is not in the document]
Comment (lorenzo-cavazzi) [reply client/src/styles/renku_additional_classes.scss:1]: Moved to https://github.com/SwissDataScienceCenter/renku-ui/pull/3327

**Review by leafty — APPROVED** [review]
LGTM

**Pull request merged**